tests/support/mod.rs

```diff
@@ -60,6 +68,33 @@ pub fn transform(suffix: &str, increase: f64) -> MockTransformConfig {
     MockTransformConfig::new(suffix.to_owned(), increase)
 }
 
+/// Creates a file with given content
+pub fn create_file(config: &str) -> PathBuf {
+    let path = temp_file();
+    overwrite_file(path.clone(), config);
+    path
+}
+
+/// Overwrites file with given content
+pub fn overwrite_file(path: PathBuf, config: &str) {
+    let mut file = OpenOptions::new()
+        .create(true)
+        .write(true)
+        .truncate(true)
+        .open(path)
+        .unwrap();
+
+    file.write_all(config.as_bytes()).unwrap();
+    file.flush().unwrap();
+    file.sync_all().unwrap();
+}
+
+pub fn create_directory() -> PathBuf {
+    let path = temp_dir();
+    Command::new("mkdir").arg(path.clone()).assert().success();
```

Comment:
Why don't we create a dir using the https://doc.rust-lang.org/std/fs/fn.create_dir.html here?

Reply:
I see, it's copied over, but it's odd either way. There are no comments on why it's like this, maybe it's just an oversight.

Reply:
Yep `create_dir` is better.